keyboards/basekeys/slice/rev1_rgb/rev1.c

```diff
@@ -0,0 +1 @@
+#include "rev1.h"
```

Comment:
this file should match the folder name rev1_rgb.c

Reply:
Complete